main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"github.com/kataras/golog"
+	"github.com/vitwit/go-fass/openfass-go"
+)
+
+func main() {
+	cli := openfass_go.NewSendClient("admin", "9a651783248f8b61a92ae2fa02c7cb627a68e061f5cc0773822f72e0c7af077d",
+		"")
+
+	golog.Info("mmmmmm.....", cli.BaseURL, cli.Method)
+
+	res, err := cli.GetSystemFunctions("admin", "9a651783248f8b61a92ae2fa02c7cb627a68e061f5cc0773822f72e0c7af077d")
```

Comment:
can we change it a bit? Can't we initialize the client once and use that everytime? If not, create an request interceptor and add these credentials from there